taggit/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig as BaseConfig
+from django.utils.translation import ugettext_lazy as _
+
+class AppConfig(BaseConfig):
```

Comment:
In other words, just rename this to `TaggitAppConfig` and this should be good to merge